components/admin_console/admin_definition.jsx

```diff
@@ -288,46 +288,16 @@ export default {
                 component: GroupSettings,
             },
         },
-        permissions: {
-            url: 'user_management/permissions',
-            title: t('admin.sidebar.permissions'),
-            title_default: 'Permissions',
-            isHidden: it.either(
-                it.isnt(it.licensed),
-                it.isnt(it.licensedForFeature('CustomPermissionsSchemes'))
-            ),
-            searchableStrings: [
-                'admin.permissions.documentationLinkText',
-                'admin.permissions.teamOverrideSchemesNoSchemes',
-                'admin.permissions.loadMoreSchemes',
-                'admin.permissions.introBanner',
-                'admin.permissions.systemSchemeBannerTitle',
-                'admin.permissions.systemSchemeBannerText',
-                'admin.permissions.systemSchemeBannerButton',
-                'admin.permissions.teamOverrideSchemesTitle',
-                'admin.permissions.teamOverrideSchemesBannerText',
-                'admin.permissions.teamOverrideSchemesNewButton',
-            ],
-            schema: {
-                id: 'PermissionSchemes',
-                component: PermissionSchemesSettings,
-            },
-        },
         systemScheme: {
-            url: 'user_management/system_scheme',
-            title: t('admin.sidebar.systemScheme'),
-            title_default: 'System Scheme',
-            isHidden: it.either(
-                it.isnt(it.licensed),
-                it.isnt(it.licensedForFeature('CustomPermissionsSchemes'))
-            ),
+            url: 'user_management/permissions/system_scheme',
+            isHidden: it.either(it.isnt(it.licensed)),
```

Comment:
You are changing here when this is visualized. There are 3 possible cases, you don't have license, so you don't see any permissions related option, you have an E10 license (not an E20 license, which is the one which has the 'CustomPermissionsSchemes' feature), so you are able to see the system schemes. Or you have an E20 license, and then you can see the System Schemes and the team schemes.

Reply:
That definitely looks like a typo. Let me inspect this closer.

Reply:
In my next commit I've removed the unnecessary "or" condition--but as far as visibility of the "System Scheme" and "Team Scheme"--it is aligned to what you've described. The change is that accessing the System Scheme is done through the "Permissions" page and not via the LHS nav. @wiersgallak has tested this as well.